examples/02_async/tsconfig.json

```diff
@@ -8,6 +8,16 @@
     "skipLibCheck": true,
     "noUncheckedIndexedAccess": true,
     "exactOptionalPropertyTypes": true,
-    "jsx": "react-jsx"
-  }
+    "jsx": "react-jsx",
+    "outDir": "./dist",
+    "tsBuildInfoFile": "./dist/.tsbuildinfo",
+    "composite": true,
+    "types": ["react/experimental"]
+  },
+  "include": ["./src"],
+  "references": [
+    {
+      "path": "./tsconfig.node.json"
+    }
+  ]
```

Comment:
Why do we need this in 02_async, not in 01_counter?
I do not understand why two tsconfig files are required, instead of one.

Reply:
because `vite.config.ts` is actually in a different environment with files in `./src`.

For example, you cannot include src files in config because of two ts configs with different files included

[tsconfig.json](https://github.com/dai-shi/waku/pull/132/files/5ee1aed268049e10f0007917dfb5cd7c66480414#diff-d77d626902deb144fd543b56199c8719c93b7f43b024d6d92f5fce182cc97551)

This is good for mono repo because sometimes you will mistakenly include files across the packages

Also it will be compiled to the different outputDir based on tsconfig.

01_counter doesn't have two configs because there is no vite.config.ts

Reply:
Thanks for the explanation. We will revisit this later.